app/controllers/medications_controller.rb

```diff
@@ -45,11 +45,24 @@ def edit
     redirect_to_medication(@medication)
   end
 
+  def add_refill(medicine)
```

Comment:
We should also write a unit test for this function!

Reply:
Hi @julianguyen, I am not quite familiar with writing tests for such a function. Would be great if someone could guide me through it!

Reply:
What I would do this update this part of the [test](https://github.com/ifmeorg/ifme/blob/master/spec/controllers/medications_controller_spec.rb#L126) to ensure that `@medication.refill` is the expected value.